### PR TITLE
OPSEXP-1651 Use existing secret to configure ACS postgres credentials

### PR DESCRIPTION
**Pull request by gionn**
Ref: OPSEXP-1651

- [x] support existing secret for acs db
- [x] document support for existing secret in bitnami chart
- [x] add unit tests

### CODE REVIEW

**Review by alxgomz — COMMENTED** [review]
again, we could use `envFrom` to inject the whole secret

**Comment by gionn** [comment]
> again, we could use `envFrom` to inject the whole secret

can't use `envFrom` because secret contain keys like `DATABASE_PASSWORD` while the container requires `SPRING_DATASOURCE_PASSWORD`

File: helm/alfresco-content-services/tests/secret-database_test.yaml [hunk 1]
```diff
@@ -0,0 +1,35 @@
...
+  - it: should have secrets set with default values
+    values: &testvalues
+      - values/test_values.yaml
+    asserts:
+      - equal:
+          path: data.DATABASE_USERNAME
+          value: YWxmcmVzY28=
+      - equal:
+          path: data.DATABASE_USERNAME
+          value: YWxmcmVzY28=
```
Comment (alxgomz) [inline helm/alfresco-content-services/tests/secret-database_test.yaml:15]: dup?

File: helm/alfresco-content-services/tests/secret-database_test.yaml [hunk 2]
```diff
@@ -0,0 +1,35 @@
...
+      database:
+        user: mycustomadmin
+        password: mycustompassword
+    asserts:
+      - equal:
+          path: data.DATABASE_USERNAME
+          value: bXljdXN0b21hZG1pbg==
+      - equal:
+          path: data.DATABASE_USERNAME
+          value: bXljdXN0b21hZG1pbg==
```
Comment (alxgomz) [inline helm/alfresco-content-services/tests/secret-database_test.yaml:35]: dup?

**Review by alxgomz — CHANGES_REQUESTED** [review]

File: helm/alfresco-content-services/tests/secret-database_test.yaml (hunk 1 above)
Comment (gionn) [reply helm/alfresco-content-services/tests/secret-database_test.yaml:15]: yes 🤦🏻

**Review by alxgomz — APPROVED** [review]

**Review by alxgomz — APPROVED** [review]

**Pull request merged**